scripts/build_windows.bat

```diff
@@ -0,0 +1,10 @@
+:: Workaround for https://github.com/conda/conda-build/issues/636
+set "PYTHONIOENCODING=UTF-8"
+
+:: SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+:: /E:ON and /V:ON options are not enabled in the batch script interpreter
+:: See: http://stackoverflow.com/a/13751649/163740
+set "CMD_IN_ENV=call cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
```

Comment:
That's a very interesting data point. Thanks for sharing, @mwcraig. I'd be curious if you could try this way of building on Windows/AppVeyor for that same use case to see if this works or needs more tweaks.